src/test/java/complete_task_use_case/CompleteTaskInteractorTest.java

```diff
@@ -0,0 +1,25 @@
+package complete_task_use_case;
+
+import data_access.DataAccessTask;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+/**
+ * Task Completion Interactor Test Cases.
+ * @author Fardin Faruk
+ */
+public class CompleteTaskInteractorTest {
+    public final static CompleteTaskOutputBoundary presenter = new CompleteTaskPresenter();
+    public final static CompleteTaskDataAccess dataAccess = new CompleteTaskDataAccess();
```

Comment:
final static variables should use screaming snake case

Reply:
DONE